src/app/app.module.ts

```diff
@@ -49,6 +50,7 @@ const appInitializerFactory =
     EffectsModule.forFeature([
       CountriesEffects,
     ]),
+    ValidationModule,
```

Comment:
As I remember, shared modules should be imported in specific modules, not in App